client/components/happychat/composer.jsx

```diff
@@ -11,7 +11,8 @@ import { isEmpty } from 'lodash';
 /**
  * Internal dependencies
  */
-import { sendChatMessage, setChatMessage } from 'state/happychat/actions';
+import { sendChatMessage } from 'state/happychat/actions';
+import { setChatMessage } from 'state/happychat/chat/actions';
```

Comment:
Should we also move `requestChatTranscript`, `receiveChatTranscript`, `receiveChatEvent` and `sendChatMessage` under `happychat/chat`?

Reply:
Answered [here](https://github.com/Automattic/wp-calypso/pull/18699#discussion_r144234917): essentially I think those actions mimic SocketIO events so they should be in `happychat/connection`.